build/dependencies.props

```diff
@@ -4,9 +4,13 @@
     <AspNetIntegrationTestingVersion>0.4.0-*</AspNetIntegrationTestingVersion>
     <AspNetCoreVersion>2.0.0-*</AspNetCoreVersion>
     <AutoMapperVersion>5.2.0</AutoMapperVersion>
+    <ImmutableCollectionsVersion>1.3.1</ImmutableCollectionsVersion>
     <InternalAspNetCoreSdkVersion>2.1.0-*</InternalAspNetCoreSdkVersion>
     <NETStandardImplicitPackageVersion>$(BundledNETStandardPackageVersion)</NETStandardImplicitPackageVersion>
```

Comment:
Which property?  `NETStandardImplicitPackageVersion`?  If we agree this is no longer needed, I will file a separate issue to remove.  I want to limit this PR to only what is needed to target net461.